index.html

```diff
@@ -0,0 +1,150 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Twitter Challenge</title>
+    <link rel="stylesheet" href="styles.css">
+</head>
+
+<body>
+    <!--MAIN LAYOUT-->
+    <div class="twitter-grid three-columns">
+        <div class="left-hand-menu">
+            <div class="left-box">
+                <img class="twitter-icon" src="icons/twitter.png" alt="twitter">
+            </div>
+
+            <!--LEFT BAR LAYOUT-->
+            <div class="left-box">
+                <ul>
+                    <li class="home two-columns align-center">
+                        <img class="gap-sm" src="icons/home.png" alt="home">
+                        <h3>Home</h3>
+                    </li>
+                    <li class="two-columns align-center">
+                        <img class="gap-sm" src="icons/magnifying-glass.png" alt="magnifying-glass">
+                        <h3>Explore</h3>
+                    </li>
+                    <li class="two-columns align-center">
+                        <img class="gap-sm" src="icons/bell.png" alt="bell">
+                        <h3>Notifications</h3>
+                    </li>
+                    <li class="two-columns align-center">
+                        <img class="gap-sm" src="icons/email.png" alt="email">
+                        <h3>Messages</h3>
+                    </li>
+                    <li class="two-columns align-center">
+                        <img class="gap-sm" src="icons/bookmark.png" alt="bookmark">
+                        <h3>Bookmarks</h3>
+                    </li>
+                    <li class="two-columns align-center">
+                        <img class="gap-sm" src="icons/account.png" alt="account">
+                        <h3>Profile</h3>
+                    </li>
+                    <li class="two-columns align-center">
+                        <img class="gap-sm" src="icons/more.png" alt="more">
+                        <h3>More</h3>
+                    </li>
+                    <button class="gap-sm left-btn">Tweet</button>
+                </ul>
+            </div>
+
+            <div class="left-box three-columns align-center">
+                    <img class="picture-sm" src="icons/anica.png" alt="anica">
+                    <p class="text-name">Anica Rahman<br>
+                    @Solo Ann</p>
+                    <img class="more-icon" src="icons/more.png" alt="more">
+                </div>
+            </div>
+
+            <!--MIDDLE SECTION LAYOUT-->
```

Comment:
Nice that you utilized comments in between... it's easier to understand which code is for what section